nuxeo-ai-model/src/main/java/org/nuxeo/ai/model/export/DatasetExportServiceImpl.java

```diff
@@ -262,9 +262,10 @@ protected void validateParams(String nxql, Collection<String> inputProperties,
     protected void getValidStats(List<PropertyType> featuresWithType, List<Statistic> stats, NxQueryBuilder qb) {
         for (PropertyType prop : featuresWithType) {
             String propName = prop.getName();
-            if (IMAGE_TYPE.equals(prop.getType()) || TEXT_TYPE.equals(prop.getType())) {
+            if (IMAGE_TYPE.equals(prop.getType())) {
                 // qb.addAggregate(makeAggregate(AGG_CARDINALITY, contentProperty(propName), EMPTY_PROPS));
             } else {
+                // Apply aggregates for Text and Category data types
```

Comment:
I wonder if it is not better really define that it is for Text and Categories, in case we have new types in the future 
(and we might have numeric classes)

Reply:
yep, sure